src/style.css

```diff
@@ -0,0 +1,135 @@
+html {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.block-product {
+  margin: 0;
+}
+
+.block-product__screen {
+  width: 200px;
+  height: 406px;
+  background-color: white;
+  font-family: Roboto , sans-serif;
+}
+
+.image {
+  position: relative;
+  top: 33px;
+  left: 20px;
+  width: 160px;
+  height: 134px;
+}
+
+.text {
+  padding: 0;
+  margin: 0;
+}
+
+.list {
+  position: relative;
+  display: flex;
+  left: 17px;
+  top: 69px;
+  width: 166px;
+  list-style: none;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product {
+  position: relative;
+  display: flex;
+  top: 73px;
+  left: 17px;
+  list-style: none;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.reviews {
+  position: relative;
+  top: 75px;
+  left: 131px;
+  list-style: none;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.price {
+  position: relative;
+  top: 99px;
+  left: 17px;
+  list-style: none;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.mony {
+  position: relative;
+  top: 81px;
+  left: 104px;
+  list-style: none;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.link {
+  position: relative;
+  top: 49px;
+  right: 12px;
+  margin: 0;
+}
```

Comment:
do not use relative position for all blocks and then position them with `top` and so on. Your blocks should be positioned by themselves (every block element is positioned on new line)